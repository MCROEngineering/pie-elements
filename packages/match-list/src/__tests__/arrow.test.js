import * as React from 'react';
import { shallow } from 'enzyme';
import { Arrow } from '../arrow';

describe('Arrow', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Arrow classes={{}} />);
  });

  describe('render', () => {
    it('renders correctly for default direction', () => {
      expect(wrapper).toMatchSnapshot();
    });

    it('renders correctly for direction left', () => {
      wrapper.setProps({ direction: 'left' });
      expect(wrapper).toMatchSnapshot();
    });
  });
});
