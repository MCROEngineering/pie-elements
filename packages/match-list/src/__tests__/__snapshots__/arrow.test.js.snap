// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow render renders correctly for default direction 1`] = `
<div
  style={
    Object {
      "transform": "rotate(180deg)",
    }
  }
>
  <pure(ArrowDropDownIcon)
    style={
      Object {
        "color": "#979797",
        "fontSize": 40,
        "transform": "rotate(90deg)",
      }
    }
  />
  <span
    className="undefined"
  />
</div>
`;

exports[`Arrow render renders correctly for direction left 1`] = `
<div
  style={Object {}}
>
  <pure(ArrowDropDownIcon)
    style={
      Object {
        "color": "#979797",
        "fontSize": 40,
        "transform": "rotate(90deg)",
      }
    }
  />
  <span
    className=""
  />
</div>
`;
